modules/Gallery/DownloadLink.tsx

```diff
@@ -24,10 +26,14 @@ export function DownloadLink({ localeCode, href }: Props) {
 
     return (
         <ButtonLink
+            title={disabled ? 'Temporarily disabled due to system maintenance' : undefined}
             variation="primary"
             forceRefresh
-            href={href}
-            className={styles.link}
+            // eslint-disable-next-line no-script-url
+            href={disabled ? 'javascript:void(0)' : href}
```

Comment:
Temporarily ofcourse, needed to keep the button interactive still to show the "title" on hover.